intrusive_list.h

```diff
@@ -76,9 +76,11 @@ class intrusive_list_base
 	void pop_back()		{ unlink(m_root.prev); }
 	void pop_front()	{ unlink(m_root.next); }
 
+	// NOTE: commenting `intrusive_list_base::size()` method declaration; missing definition ~SK
 	// @note: allow for constant complexity way of checking this
 	// (at a cost of additional variable)?
-	size_type size() const;
+	//size_type size() const;

```

Comment:
@msinilo is this declaration serving as a concept or "template guide" for classes deriving from it? If so, I'll revert this change. See also the same instance in intrusive_slist.h.

Reply:
Definition is in the .cpp file (base is not a template)

Reply:
Well, this is kind of awkward...reverting noob mistake lol. Thanks.